src/providers/ad/ad_gpo.c

```diff
@@ -4742,9 +4742,18 @@ static void gpo_cse_done(struct tevent_req *subreq)
     ret = ad_gpo_parse_gpo_child_response(state->buf, state->len,
                                           &sysvol_gpt_version, &child_result);
     if (ret != EOK) {
-        DEBUG(SSSDBG_CRIT_FAILURE,
-              "ad_gpo_parse_gpo_child_response failed: [%d][%s]\n",
-              ret, sss_strerror(ret));
+        if (ret == EINVAL) {
+            DEBUG(SSSDBG_CRIT_FAILURE,
+                  "ad_gpo_parse_gpo_child_response failed: [%d][%s]\n "
```

Comment:
I think it's better to avoid `\n` inside debug message.

Reply:
To be precise here: `\n` should be only at the end of debug message.